src/index.html

```diff
@@ -1,18 +1,75 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag should not be self-closing. Remove the `/` at the end of the tag to properly open the `<html>` element.